### DISCUSSION TITLE
过年好

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3296] [讨论]
🎆🧨🎆🎇🎇🎆🧨🧨🧨

**Comment by allMagicNB** [comment]
女子 女子 女子

**Comment by zkitefly** [comment]
<image src="https://github.com/Hex-Dragon/PCL2/assets/64117916/e2c78094-06d6-4354-93d6-75becd72cb48" width="200"/>

**Reply by hejiehao** [reply]
你的手机号捏？

**Reply by iPodtouch6557** [reply]
我这的雪七天都没化
（吗的我们中秋有烟花春节反而没有）

**Comment by iPodtouch6557** [comment]
我们的短信提示（）
![IMG_9752](https://github.com/Hex-Dragon/PCL2/assets/118656946/1b4c46e3-4605-4f13-9d59-66d5c4b12764)
![IMG_9751](https://github.com/Hex-Dragon/PCL2/assets/118656946/3696a597-d3f5-470a-a949-92c74ff65e0c)

**Reply by iPodtouch6557** [reply]
我是在一年过年学会剥瓜子的（

**Comment by zkitefly** [comment]
![Image_1707492296746.jpg](https://github.com/Hex-Dragon/PCL2/assets/64117916/5ae315b5-dd7d-4e8e-9a49-5c69827d7856)

**Reply by iPodtouch6557** [reply]
最垃圾节目没有之一

**Reply by Deep-Dark-Forest** [reply]
据我所知你所在的省份好像是春晚收视率倒数的（
~~然后你给你们省贡献了收视率~~

**Reply by iPodtouch6557** [reply]
没有，我过年都在山东乡下

**Comment by zkitefly** [comment]
新年快乐

**Reply by WForst-Breeze** [reply]
新年快乐

**Comment by allMagicNB** [comment]
新年快乐

**Comment by SuoHouDaiShi** [comment]
耶咦耶咦耶咦耶哦哦\~耶咦耶咦耶咦耶哦哦\~我恭喜你发财~我恭喜你金彩\~（

**Reply by iPodtouch6557** [reply]
宛平南路600号欢迎你（）
话说我第一次知道大部分地区初二回娘家，我这都初三
